corehq/apps/app_manager/models.py

```diff
@@ -901,13 +901,17 @@ class FormSchedule(DocumentSchema):
     termination_condition = SchemaProperty(FormActionCondition)
 
 
-class CustomAssertion(DocumentSchema):
+class Assertion(DocumentSchema):
+    test = StringProperty()
+    text = DictProperty(StringProperty)
```

Comment:
Should this be a `LabelProperty`?

Reply:
Maybe?  I was torn because I'm not sure what the difference between a `DictProperty()` and a `DictProperty(StringProperty)` is in practice - do we have wacky data that doesn't match and this'll cause 500s on some apps?  If I make `Assertion` and `CustomAssertion` entirely separate then there's no issue, though.

Reply:
ah, at the time I wrote this I was thinking that the assertion model was new in this PR. Old apps have all kinds of bizarre config, I think it's fine to not poke at that particular rabbit hole.